packages/p2p/src/components/my-ads/my-ads.jsx

```diff
@@ -18,16 +19,17 @@ class MyAds extends Component {
     };
 
     render() {
+        const { is_authenticated, is_active } = this.context.agent_info;
         return (
             <div className='p2p-my-ads'>
                 {this.state.show_form ? (
                     <FormAds ad_id={this.state.ad_id} handleShowForm={this.handleShowForm} />
                 ) : (
                     <Fragment>
                         <div className='p2p-my-ads__header'>
-                            {/* <ToggleAds
-                                is_enabled={true}
-                            /> */}
+                            {is_authenticated ? <ToggleAds
```

Comment:
why do we need to check is_authenticated?

Reply:
I think there is still a discussion going on about what to do when is_authenticated is false. IMO we can ignore adding it for now until the decision is made.

Reply:
in the Backend card *note there is a saying "We don't allow updating agent information when the agent is not authenticated."

Reply:
I think we need a design for that, better show it for now and show an error message on toggle. since edit/delete will also fail, better show a message to the user instead of hiding their actions

Reply:
yes, i have added todo for design on error message when toggling